server/src/domain/system-config/system-config.core.ts

```diff
@@ -128,7 +133,11 @@ export class SystemConfigCore {
 
   public config$ = singleton;
 
-  constructor(private repository: ISystemConfigRepository) {}
+  public schedulerRegistry: SchedulerRegistry;
+
+  constructor(private repository: ISystemConfigRepository) {
+    this.schedulerRegistry = new SchedulerRegistry();
+  }
```

Comment:
A few things here. We should not create the registry class directly, we should just inject it into the constructor. Also, we try to only inject repositories that can easily be mocked and are technology independent. I'm not 100% sure we need a repository for a basic set interval though. I'd be fine using it directly. It's only setup and access in one place anyway, so using the scheduler seems overly complicated. I think this is the wrong class too. It should be implemented in server-info.service probably, since this class is all about storage and retrieval of configuration not what they're used for.